src/style.css

```diff
@@ -1 +1,111 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
```

Comment:
```suggestion
* {
  box-sizing: border-box;
}
```

Reply:
when you perform this task, the search icon disappears

Reply:
it's the better way to set box-sizing to avoid duplication (generally it's an approach so you want to stick across the entire application)

not critical maybe, but if you have some problems in this case - try to debug (use devTools), ask in the chat if you need help